spec/models/project_services/hipchat_service_spec.rb

```diff
@@ -36,6 +36,8 @@
         project_id: project.id,
         project: project,
         room: 123456,
+        color: 'yellow',
+	notification: '0',
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Tab detected.